lib/dbl.php

```diff
@@ -129,28 +138,25 @@ function sanitized_dsn() {
 
     /** @return ?\mysqli */
     function connect() {
-        assert(($this->name ?? "") !== "");
-        if ($this->socket) {
-            $dblink = new mysqli($this->host, $this->user, $this->password, "", $this->port, $this->socket);
-        } else if ($this->port !== null) {
-            $dblink = new mysqli($this->host, $this->user, $this->password, "", $this->port);
-        } else {
-            $dblink = new mysqli($this->host, $this->user, $this->password);
-        }
+	assert(($this->name ?? "") !== "");
+
+        $dblink = new mysqli();
+        $dblink->ssl_set($this->sslKey, $this->sslCert, $this->sslCaCert, null, null);
```

Comment:
I would prefer that all these options be encapsulated under a bool $Opt["dbSsl"].

Reply:
done.